xbmc/addons/IAddon.h

```diff
@@ -92,6 +92,7 @@ namespace ADDON
     virtual bool GetSettingString(const std::string& key, std::string& value) = 0;
     virtual CAddonSettings* GetSettings() const =0;
     virtual const ADDONDEPS &GetDeps() const =0;
+    virtual int NoDeps(std::map<std::string,std::string>& added) const = 0;
```

Comment:
I can consider a wrapper but your name suggestion says read code again;) No is englisj abbrevation of number, ie this counts deps recursively.